### PR TITLE
create plugin for network information from cordova

### PR DESCRIPTION
**Pull request by ndcunningham**
#2 
Looking for a feedback if the implementation is towards what you are looking for.

### CODE REVIEW

**Comment by ndcunningham** [comment]
Looking for a feedback if the implementation is towards what you are looking for.

**Comment by robertarnesson** [comment]
Looks great! I would just change the way the constants are defined (see example below), to make them read-only. They are still accessable the same way e.g. console.log(Constants.MY_CONSTANT)) outputs 1

class Constants {
    public static get MY_CONSTANT(): number    { return 1; }
}

**Comment by ndcunningham** [comment]
Thanks for the comments! I updated it as requested

**Comment by ndcunningham** [comment]
I have a question though. 

I followed your approach but when not do an enum of strings? 

Then we can type it back to the return value from connection type.
```
export enum Connection {
UNKNOWN = 'unknown'
}

 connectionType(): Observable<Connection> {
    return Cordova.deviceready.mergeMap(() =>
      ZoneObservable.of(this.zone, (<any>window).navigator.connection.type)
    );
  }
```
Syntax is the same. Same rules applies,  *readonly* but gives you typing info for one of the methods

**Comment by robertarnesson** [comment]
Yes I like this. we could replace all constants with enum

**Comment by ndcunningham** [comment]
Updated Let me know if which others you will update

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 "use strict";
-var Observable_1 = require('rxjs/Observable');
-require('rxjs/add/operator/share');
+exports.__esModule = true;
```
Comment (ndcunningham) [inline index.js]: I didn't realise this. It didn't show up in my vscode. Is this a problem?

Comment (robertarnesson) [reply index.js]: strange.. yeah this will not work. if you compile with npm run build these things should not happen I think

Comment (ndcunningham) [reply index.js]: Updated. i was using `tsc plugin/network-information.ts`

**Pull request merged**